slices/apparmor.yaml

```diff
@@ -0,0 +1,199 @@
+package: apparmor
+
+essential:
+  - apparmor_copyright
+
+slices:
+  # Also relies on debconf, however it is solely used by install hook
+  # to determine if any non default homedir has been configured.
+  bins:
+    essential:
+      - apparmor_config
+      - apparmor_profiles
+      - libc6_libs
+      - lsb-base_bins
+    contents:
+      /etc/init.d/apparmor:
+      /sbin/apparmor_parser:
+      /usr/bin/aa-enabled:
+      /usr/bin/aa-exec:
+      /usr/bin/aa-features-abi:
+      /usr/sbin/aa-load:
+      /usr/sbin/aa-remove-unknown:
+      /usr/sbin/aa-status:
+      /usr/sbin/aa-teardown:
+      /usr/sbin/apparmor_status:
+
+  config:
+    contents:
+      /etc/apparmor/parser.conf:
+      /usr/lib/sysctl.d/10-apparmor.conf:
+
+  extras:
+    contents:
+      /lib/apparmor/apparmor.systemd:
+      /lib/apparmor/profile-load:
+      /lib/apparmor/rc.apparmor.functions:
+      /usr/lib/systemd/system/apparmor.service:
+
+  profiles:
+    contents:
+      /etc/apparmor.d/1password:
+      /etc/apparmor.d/Discord:
+      /etc/apparmor.d/MongoDB_Compass:
+      /etc/apparmor.d/QtWebEngineProcess:
+      /etc/apparmor.d/abi/3.0:
+      /etc/apparmor.d/abi/4.0:
+      /etc/apparmor.d/abi/kernel-5.4-outoftree-network:
+      /etc/apparmor.d/abi/kernel-5.4-vanilla:
+      /etc/apparmor.d/abstractions/**:
```

Comment:
could there ever be another pkg bringing profiles into this path? if so, please consider future conflicts.

Reply:
I found a package with conflicts, good point!